.github/workflows/github-actions-manual-update-rules.yml

```diff
@@ -0,0 +1,55 @@
+name: Manually Trigger Update Rules
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'Type of update (overwrite or normal)'
+        required: true
+        default: 'normal'
+      commitsha:
+        description: 'Commit SHA'
+        required: true
```

Comment:
Users should not have to type the commit sha. We should just get the last sha of the branch we are running on.